qa-acceptance-testing/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM teracy/ubuntu:18.04-dind-18.09.9
```

Comment:
We recently modified it to just use `ubuntu:18.04` (see [here](https://github.com/mendersoftware/mender-qa/blob/master/.gitlab-ci.yml#L380), from https://github.com/mendersoftware/mender-qa/pull/442); please modify here accordingly.